src/config/inky.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+return [
+    /*
+    |--------------------------------------------------------------------------
+    | Email Stylesheets
+    |--------------------------------------------------------------------------
+    |
+    | Here you may specify any CSS files you would like to apply to your emails.
+    | These CSS files must be placed in your public directory.  Please include
+    | the path starting with the 'public' directory.  You do not need to include
+    | any <link> elements for these CSS files in your .inky.php templates.
+    |
+    */
+    'stylesheets' => [
+        'public/css/foundation-emails.css',
```

Comment:
Please update the paths and update the comments to remove references to `public`.